noir-projects/noir-contracts/contracts/test_contract/src/main.nr

```diff
@@ -202,6 +202,18 @@ contract Test {
         context.message_portal(note.get_commitment());
     }
 
+    #[aztec(public)]
+    fn create_l2_to_l1_message_arbitrary_recipient_public(content: Field, recipient: EthAddress) {
+        // Public oracle call to emit new commitment.
+        context.message_recipient(recipient, content);
+    }
+
+    #[aztec(public)]
```

Comment:
This function is public should probably be prive for the test to check both.

Reply:
This was a mistake. Good catch. Addressed in https://github.com/AztecProtocol/aztec-packages/pull/4512/commits/22c54ce41364fa47db1146660a46fe73c60b1b72